src/cmakeFindModules/FindOpenImageIO.cmake

```diff
@@ -0,0 +1,18 @@
+#find headers
```

Comment:
it's already a dependency :-)
But it cannot write. Well we could add the writing part to that...